### PR TITLE
[CI] GitHub Actions support for Linux platform

### PR DESCRIPTION
**Pull request by shahmishal**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2024 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+#
+
```
Comment (bnbarham) [inline CODEOWNERS:10]:
```suggestion

* @ahoppen @allevato @bnbarham @shawnhyam

```

**Review by bnbarham — APPROVED** [review]
🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉

**Comment by allevato** [comment]
Exciting! 🎉

**Comment by ahoppen** [comment]
Wohoooo. This is exciting 🥳

**Pull request merged**